syntax/blocks.js

```diff
@@ -121,6 +121,7 @@ var allBlocks = scratchCommands.map(function(def) {
 
 var unicodeIcons = {
   "@greenFlag": "⚑",
+  "@stopSign": "⯃",
```

Comment:
Hmm, putting in the unicode doesn't seem to work for me: it doesn't get recognised as the stop sign. :thinking:

Reply:
Ah, I think this list is only used for translation shenanigans. We'll only "recognise" an icon if it appears inside a real block like `when ⚑ clicked`, I think.

Probably best to remove this for now?